src/node_file.h

```diff
@@ -55,6 +55,35 @@ enum class FsStatFsOffset {
 constexpr size_t kFsStatFsBufferLength =
     static_cast<size_t>(FsStatFsOffset::kFsStatFsFieldsNumber);
 
+// the possible file extensions that should be tested
+// 0-6: when packageConfig.main is defined
+// 7-9: when packageConfig.main is NOT defined,
+//      or when the previous case didn't found the file
+const std::vector<std::string> legacy_main_extensions = {"",
```

Comment:
This can be a `std::array` since this variable is not mutable. Than we can convert this to `constexpr std::array`